source/Ship.h

```diff
@@ -287,6 +287,10 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double Health() const;
 	// Get the hull fraction at which this ship is disabled.
 	double DisabledHull() const;
+	// Get the (absolute) amount of hull that needs to be damaged until the
+	// ship becomes disabled, returns 0 if the ships hull is already below the
+	// disabled level.
+	double HullUntilDisabled() const;
```

Comment:
Since this is returning a level instead of a fraction, I'd still move it below ShieldLevel so that the fractions (Shield, Hull, Health, DisabledHull) and the levels (ShieldLevel, HullUntilDisabled) are ordered the same way (Shield, then Hull, then Disabled).